packages/lb-annotation/src/core/pointCloud/index.ts

```diff
@@ -299,6 +300,43 @@ export class PointCloud {
     }, {});
   }
 
+  /**
+   * add new sphere just like adding new box
+   * @param sphereParams
+   */
+  public AddSphereToSense = (sphereParams: IPointCloudSphere, color = 'blue') => {
+    const id = sphereParams.id ?? uuid();
+
+    this.removeObjectByName(id);
+
+    const { center, widthSegments, heightSegments, radius } = sphereParams;
+    const group = new THREE.Group();
+    const spGeo = new THREE.SphereGeometry(radius, widthSegments, heightSegments);
```

Comment:
Maybe it is not need to record `widthSegments` & `heightSegments` ？

Reply:
Just in case someday there will be a requirement for customized sphere style?